docs/SchemaMigrationGuide.md

```diff
@@ -66,24 +67,19 @@ There is a configurable property in the launchSettings.json which can be set to
 
 `SqlServer:SchemaOptions:AutomaticUpdatesEnabled": "true"`
 
-* If the property sets to 'true', then the schema would be upgraded automatically on the server startup.
-* If the property sets to 'false', then the schema would remain in the same state. Moreover, the schema would only be upgraded by running the schema migration tool as needed.
+* If the property sets to 'true', then the SQL Server schema would be upgraded automatically on the server startup.
+* If the property sets to 'false', then the SQL Server schema would remain in the same state. Moreover, the schema would only be upgraded by running the schema migration tool as needed.
 
 This configurable property should be considered for each web project
 
-* Stu3 - https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Stu3.Web/Properties/launchSettings.json#L27
-* R4 - https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.R4.Web/Properties/launchSettings.json#L28
-* R5 - https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.R5.Web/Properties/launchSettings.json#L28
-
  Note: This guide contains generalized advice and may not take your specific environment or application into account.
 
  ## Frequently Asked Questions
 
  Q1. What is upgraded first, app service or database schema?
 
- Ans. The app service is upgraded first. Because firstly it contains the scripts to migrate the schema and secondly the code is considered to work for a number of schema versions and will specify the min and max supported versions.
+ A1. The FHIR Server for Azure app service is upgraded first. Because firstly it contains the scripts to migrate the SQL Server schema and secondly the code is considered to work for a number of SQL Server schema versions and will specify the min and max supported versions.
 
- Q2. What should we do in case of automatic schema upgrade fails?
+ Q2. What should we do in case of automatic SQL Server schema upgrade fails?
 
- Ans. If automatic schema upgrade fails and application is unable to start then you can toggle the schema auto upgrade flag as mentioned under the section (How to toggle between the options) so that the upgrade scripts would not run and application would start on the current version.
- Also, you can reach out to the concerned team with the error description to get further help on the issue.
+ A2. If automatic SQL Server schema upgrade fails and web application is unable to start then you can toggle the SQL Server schema auto upgrade flag as mentioned under the section (How to toggle between the options) so that the upgrade scripts would not run and application would start on the current version.
```

Comment:
Very minor but: `If an automatic` and `the web application`

Reply:
I believe you might also be able to put a hyperlink to the "How to toggle between the options" section of the document if you are feeling like being a documentation nerd like me 😎